inst/@sym/sym.m

```diff
@@ -226,42 +225,53 @@
 
   elseif (isa (x, 'double'))  % Handle doubles
     if (~isreal (x))
-      s = sym (real (x)) + sym ('i')*sym (imag (x));
-      return
-
+      xx = {real(x); imag(x)};
```

Comment:
Cannot use spaces between function name and parenthesis here as it leads to error...